src/vs/workbench/services/textfile/test/textFileService.io.test.ts

```diff
@@ -186,7 +186,7 @@ suite('Files - TextFileService i/o', () => {
 		assert.equal(detectedEncoding, UTF8);
 	});
 
-	test('create - UTF 8 BOM - empty content - snapshot', async () => {
+	test.skip('create - UTF 8 BOM - empty content - snapshot', async () => { // {{SQL CARBON EDIT}} skip test
```

Comment:
I don't think we should be skipping this entirely - if you add the @UNSTABLE@ tag it will then not run for the stable test runs but we'll still be able to easily run it and see the history if we want to revisit enabling this in the future.

Reply:
Do we actually use the @UNSTABLE@ tag?

Reply:
Yes. By default test.bat will only run stable tests :

https://github.com/microsoft/azuredatastudio/blob/effa50a9bd107e1ce3c9a397f776dedce33244d9/scripts/test.bat#L19

And then there's test-unstable.bat for running unstable tests :

https://github.com/microsoft/azuredatastudio/blob/effa50a9bd107e1ce3c9a397f776dedce33244d9/scripts/test-unstable.bat

Reply:
Should you be doing the same for https://github.com/microsoft/azuredatastudio/pull/7882/files ?

Reply:
I didn't realize we added support for the unstable tag here, we're skipping a bunch of tests elsewhere with .skip, so we should look into that and add them there.

Reply:
I agree, before today we hadn't decided to do that but I think it'd be useful. Although there's some tests for features we removed on purpose that can stay as .skip (we'll have to triage on a case-by-case basis)